src/arrayMethodSort.js

```diff
@@ -4,8 +4,28 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
+  [].__proto__.sort2 = function (compareFunction) {
     // write code here
+    for (let i = 0; i < this.length; i++) {
+      for (let j = 0; j < this.length - 1 - i; j++) {
+        let result;
+
+        if (compareFunction === undefined) {
+          result = this[j].toString() > this[j + 1].toString() ? 1 : 0;
```

Comment:
you don't need this check, just add the default value for compareFunction argument